sirbot-pyslackers/endpoints/slack/commands.py

```diff
@@ -9,16 +9,32 @@
 
 def create_endpoints(plugin):
     plugin.on_command("/admin", tell_admin)
-    plugin.on_command("/howtoask", ask)
+    plugin.on_command("/howtoask", how_to_ask)
     plugin.on_command("/gif", gif_search)
+    plugin.on_command("/justask", just_ask)
     plugin.on_command("/pypi", pypi_search)
     plugin.on_command("/sponsors", sponsors)
     plugin.on_command("/snippet", snippet)
     plugin.on_command("/report", report)
     plugin.on_command("/save", save_conversation)
 
 
-async def ask(command, app):
+async def just_ask(command, app):
+    slack = app.plugins["slack"]
+    response = Message()
+    response["channel"] = command["channel_id"]
+    response["unfurl_links"] = False
```

Comment:
This is not necessary. By default it's at `True` but in this case there is no link in the message, so i can be left off.

Reply:
good catch.

Reply:
I left that in due to adding a link as shown by wmont  in #community_projects